pywinauto/controls/HwndWrapper.py

```diff
@@ -1223,6 +1223,76 @@ def scroll(self, direction, amount, count = 1, retry_interval = None):
     # Non PEP-8 alias
     Scroll = scroll
 
+   #-----------------------------------------------------------
+    def get_scroll_info(self, hwnd, fnBar, lpsi):
+        win32functions.GetScrollInfo(hwnd, fnBar, lpsi)
+        if not lpsi:
+            return False
+        else:
+            return True
+
+    # Non PEP-8 alias
+    GetScrollInfo = get_scroll_info
+
+    #-----------------------------------------------------------
+    def get_scroll_pos(self, hwnd, nBar):
+        scroll_pos = win32functions.GetScrollPos(hwnd, nBar)
+        if not scroll_pos:
+            return 0
+        else:
+            return scroll_pos
+
+    # Non PEP-8 alias
+    GetScrollPos = get_scroll_pos
+
+    #-----------------------------------------------------------
+    def get_scroll_bar_info(self, hwnd, idObject):
+        scroll_bar_info = win32structures.PSCROLLBARINFO
+        scroll_bar_info.cbSize = ctypes.sizeof(scroll_bar_info)
+        scroll_bar_info = win32functions.GetScrollBarInfo(hwnd, idObject)
+        print("scroll_bar_info:")
+        print(scroll_bar_info)
+        if scroll_bar_info:
+            return True
+        else:
+            return False
+
+    # Non PEP-8 alias
+    GetScrollBarInfo = get_scroll_bar_info
+
+    #-----------------------------------------------------------
+    def wheel_mouse(self, distance, pressed = "", coords = (0, 0), count = 1, retry_interval = None):
+        "Wheel the mouse by WM_MOUSEWHEEL"
+        WHEEL_DELTA = 120
+        coeff = distance/WHEEL_DELTA
```

Comment:
If you mean integer division, you must use `//` because Python 3 uses one slash (`/`) as a float division.